src/coreclr/nativeaot/Runtime/amd64/CachedInterfaceDispatchAot.S

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+.intel_syntax noprefix
+#include <AsmOffsets.inc>         // generated by the build from AsmOffsets.cpp
+#include <unixasmmacros.inc>
+
+// trick to avoid PLT relocation at runtime which corrupts registers
+#define REL_C_FUNC(name) C_FUNC(name)@gotpcrel
+
+// Stub dispatch routine for dispatch to a vtable slot
+LEAF_ENTRY RhpVTableOffsetDispatch, _TEXT
+        int 3
+        // UNIXTODO: Implement this function
```

Comment:
Is this meant to remain a TODO when merged? not clear to me from the previous conversation about this line